src-electron/db/query-package.js

```diff
@@ -186,6 +186,38 @@ async function getPackageByPackageId(db, packageId) {
     .then(dbMapping.map.package)
 }
 
+/**
+ * Returns a package ref from the attribute ID
+ *
+ * @param db
+ * @param attributeId
+ * @returns package ref
+ */
+async function getPackageRefByAttributeId(db, attributeId) {
+  let package_ref = await dbApi.dbAll(
+    db,
+    'SELECT PACKAGE_REF FROM ATTRIBUTE WHERE ATTRIBUTE_ID = ?',
+    [attributeId]
+  )
+  return package_ref[0].PACKAGE_REF
+}
+
+/**
+ * Returns path to the meta file from the package Id
+ *
+ * @param db
+ * @param packageId
+ * @returns Meta file path
+ */
+async function getMetaFile(db, packageId) {
```

Comment:
Use query-package#getPackageByPackageId instead and get rid of this
It returns the path
`const querySelectFromPackage = `
SELECT 
  PACKAGE_ID,
  **PATH**,
  TYPE,
  CRC,
  VERSION,
  CATEGORY,
  DESCRIPTION
FROM PACKAGE ``

Reply:
done